### PR TITLE
docs(issue): added docs for wrong detection issues

### PR DESCRIPTION
**Pull request by afdesk**
## Description

Added information about wrong detection issues to the docs.

## Checklist
- [x] I've read the [guidelines for contributing](https://github.com/aquasecurity/trivy/blob/main/CONTRIBUTING.md) to this repository.
- [x] I've followed the [conventions](https://github.com/aquasecurity/trivy/blob/main/CONTRIBUTING.md#title) in the PR title.
- [ ] I've added tests that prove my fix is effective or that my feature works.
- [x] I've updated the [documentation](https://github.com/aquasecurity/trivy/blob/main/docs) with the relevant information (if needed).
- [ ] I've added usage information (if the PR introduces new options)
- [ ] I've included a "before" and "after" example to the description (if the PR is a user interface change).

### CODE REVIEW

File: docs/community/contribute/issue.md [hunk 1]
```diff
@@ -4,3 +4,18 @@ Thank you for taking interest in contributing to Trivy!
...
+if Trivy can't detect any CVEs or shows false positive result, at first do the next steps:
+- run Trivy with `-f json` that shows data sources. Please make sure that data source is correct.
+- visit [Github Advisory Database](https://github.com/advisories) and search CVE-ID.
+- visit [Gitlab Advisory Database](https://advisories.gitlab.com/) and search CVE-ID .
+
+If the data source is correct and Trivy shows wrong results, please raise an issue on Trivy
+
+If you find a problem, it'll be nice to fix it:
+* How to contribute to a GitHub security advisory: https://github.blog/2022-02-22-github-advisory-database-now-open-to-community-contributions/
+* Create an issue to Gitlab Advisory Database: https://gitlab.com/gitlab-org/security-products/gemnasium-db/-/issues/new
```
Comment (knqyf263) [inline docs/community/contribute/issue.md:21]: `mkdocs` may show different from GitHub markdown. Please make sure that the doc is not broken by `make mkdocs-serve`.

image

File: .github/ISSUE_TEMPLATE/WRONG_DETECTION.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Please, read the documentation before creating an issue:
+https://aquasecurity.github.io/trivy/latest/community/contribute/issue/
+
+-->
+
+## Check Advisory Databases
+- [ ] run Trivy with `-f json` that shows data sources. Please make sure that data source is correct.
+- [ ] visit [Github Advisory Database](https://github.com/advisories) and search CVE-ID.
+- [ ] visit [Gitlab Advisory Database](https://advisories.gitlab.com/) and search CVE-ID.
```
Comment (knqyf263) [inline .github/ISSUE_TEMPLATE/WRONG_DETECTION.md:16]:
```suggestion
## Checklist
- [ ] Read [the documentation regarding wrong detection](https://aquasecurity.github.io/trivy/latest/community/contribute/issue/#wrong-detection)
- [ ] Confirm that a security advisory in data sources is correct
    - run Trivy with `-f json` that shows data sources. Please make sure that the security advisory is correct.
```

File: .github/ISSUE_TEMPLATE/WRONG_DETECTION.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+labels: ["kind/security-advisory"]
+about: If Trivy doesn't detect something, or shows false positive detection
+---
+<!--
+
+Please, read the documentation before creating an issue:
+https://aquasecurity.github.io/trivy/latest/community/contribute/issue/
+
+-->
+
```
Comment (knqyf263) [inline .github/ISSUE_TEMPLATE/WRONG_DETECTION.md:12]: We can move it to check list.

File: docs/community/contribute/issue.md [hunk 4]
```diff
@@ -4,3 +4,18 @@ Thank you for taking interest in contributing to Trivy!
 - Please spend a small amount of time giving due diligence to the issue tracker. Your issue might be a duplicate. If it is, please add your comment to the existing issue.
 - Remember that users might search for your issue in the future, so please give it a meaningful title to help others.
 - The issue should clearly explain the reason for opening, the proposal if you have any, and any relevant technical information.
+
+## Wrong detection
+Trivy depends on Github Advisory Database and Gitlab Advisory Database.
```
Comment (knqyf263) [inline docs/community/contribute/issue.md:9]:
```suggestion
Trivy depends on [multiple data sources](https://aquasecurity.github.io/trivy/latest/docs/vulnerability/detection/data-source/).
```

File: docs/community/contribute/issue.md [hunk 5]
```diff
@@ -4,3 +4,18 @@ Thank you for taking interest in contributing to Trivy!
...
+Sometime these databases contain mistakes.
+
+if Trivy can't detect any CVEs or shows false positive result, at first do the next steps:
```
Comment (knqyf263) [inline docs/community/contribute/issue.md:12]:
```suggestion
If Trivy can't detect any CVE-IDs or shows false positive result, at first please follow the next steps:
```

File: docs/community/contribute/issue.md (hunk 1 above)
Comment (knqyf263) [reply docs/community/contribute/issue.md:21]: I'll fix it this time.

Comment (afdesk) [reply docs/community/contribute/issue.md:21]: > I'll fix it this time.

oh, thanks. will be careful

**Review by knqyf263 — APPROVED** [review]

**Pull request merged**